Examples/Tests/Langmuir/analysis_langmuir_multi_2d.py

```diff
@@ -2,31 +2,38 @@
 
 # Copyright 2019 Jean-Luc Vay, Maxence Thevenet, Remi Lehe
 #
-#
 # This file is part of WarpX.
 #
 # License: BSD-3-Clause-LBNL
 
+# This is a script that analyzes the simulation results of the following CI tests:
+# - Langmuir_multi_2d_nodal
+# - Langmuir_multi_2d_psatd
+# - Langmuir_multi_2d_psatd_nodal
+# - Langmuir_multi_2d_psatd_momentum_conserving
+# - Langmuir_multi_2d_psatd_current_correction
+# - Langmuir_multi_2d_psatd_current_correction_nodal
+# - Langmuir_multi_2d_psatd_Vay_deposition
+# - Langmuir_multi_2d_psatd_Vay_deposition_nodal
+# - Langmuir_multi_2d_MR
```

Comment:
This should be a comprehensive list of CI tests for which this analysis script is currently used.